draft-ietf-quic-transport.md

```diff
@@ -1707,11 +1738,17 @@ controller.
 
 ### Response to a RST_STREAM
 
-Since streams are bidirectional, a sender of a RST_STREAM needs to know how many
-bytes the peer has sent on the stream.  If an endpoint receives a RST_STREAM
-frame and has sent neither a FIN nor a RST_STREAM, it MUST send a RST_STREAM in
-response, bearing the offset of the last byte sent on this stream as the final
-offset.
+RST_STREAM terminates a stream abruptly.  Whether any action or response can
+or should be taken on the data already received is an application-specific
+issue.  If the sender of a RST_STREAM wishes to explicitly state that no future
+data will be processed, that endpoint MAY send a REQUEST_RST frame at the same
+time.
+
+Regardless, it will often be the case that upon receipt of a RST_STREAM, an
+endpoint will choose to stop sending data in its own direction. An endpoint that
+receives a RST_STREAM frame MUST still close the stream for sending in the other
+direction, either with a STREAM frame containing the FIN bit or a RST_STREAM
+frame of its own, but MAY choose not to do so immediately.
```

Comment:
This doesn't really work for me.  The "MUST still close" clause is toothless in light of the "MAY choose not to" that follows it.  We should keep things like the MUST out of this.  There are many cases where it is appropriate to close your writer when your reader aborts, but that will depend a great deal on context.

I would avoid normative language in this paragraph and instead note that aborting a read might trigger the need for an abort on the write.